src/hooks/useTaskForm.js

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import { v4 as uuidv4 } from "uuid";
+
+const useTaskForm = (tasks, setTasks, handleCloseModal) => {
+    const [isFormValid, setIsFormValid] = useState(false);
+
+    const validateForm = () => {
+        return task.title !== "" && task.description !== "" && task.user !== "";
+    };
+
+    const [task, setTask] = useState({
```

Comment:
could you move default object to seperate constant

Reply:
done